gap/semigroups/semieunit.gd

```diff
@@ -0,0 +1,100 @@
+#############################################################################
+##
+#W  semieunit.gd
+#Y  Copyright (C) 2016                                    Christopher Russell
+##
+##  Licensing information can be found in the README file of this package.
+##
+#############################################################################
+##
+
+DeclareCategory("IsMcAlisterTripleSemigroupElement",
+                IsAssociativeElement and IsMultiplicativeElementWithInverse);
+
+# This is a representation for McAlister triple semigroup elements, which are
+# created via the function McAlisterTripleSemigroupElement.
+#
+# The components are:
```

Comment:
This says that there are 3 components, it should really say that there are 3 positions, and what is stored in each position, and then in `DeclareRepresentation` you only declare 2 positions, so which is it, 2 or 3?

Reply:
The components 'Vertex' and 'Group element' are accessible by [] but parent is not. Instead parent is accessed via `McAlisterTripleSemigroupElementParent` (at some point we decided it was not user friendly for it to be accessible by [] because these elements are printed as pairs, not triples).

Should there be 2 positions declared or 3?
(Is this number the positions accesible by [] or the ones accessible by ![]?)

Reply:
It should be a description of the internal representation, so the ones accessible by ![], and so it should be 2 positions, according to your description.

Reply:
There are 3 positions accessible by ![]. Did you mean to say it should be 3?